core/Symbols.cc

```diff
@@ -2201,14 +2201,16 @@ uint32_t ClassOrModule::hash(const GlobalState &gs) const {
     if (!members().empty()) {
         // Rather than use membersStableOrderSlow, which is... slow..., use an order dictated by symbol ref ID.
         // It's faster to sort, and SymbolRef IDs are stable during hashing.
+        // TODO(jez) Why is this necessary, if everything already hashes its owner?
         vector<core::SymbolRef> membersToHash;
         membersToHash.reserve(this->members().size());
         for (auto e : this->members()) {
             if (!e.second.exists()) {
                 continue;
             }
 
-            if (e.second.isMethod() && e.second.asMethodRef().data(gs)->ignoreInHashing(gs)) {
+            // if (e.second.isMethod() && e.second.asMethodRef().data(gs)->ignoreInHashing(gs)) {
```

Comment:
Is this an unwritten TODO?

Reply:
Yes thank you.

Effectively, every method is now ignored in hashing, which is what the uncommented condition now reads. I will delete this line.